ci/travis/astyle.sh

```diff
@@ -19,6 +21,13 @@ then
 	fi
 fi
 
+is_valid_file(){
+        for aign in $ASTYLE_IGNORE; do
```

Comment:
does this work for paths that include spaces ?

Reply:
Not always. I'll add this feature.